package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@datadog/native-iast-rewriter",
   "homepage": "https://github.com/DataDog/dd-native-iast-rewriter-js/blob/main/README.md",
-  "version": "0.1.0-beta.3",
+  "version": "1.0.0",
```

`pre` to merge in master, and we will change it in `v1.x` branch.
```suggestion
  "version": "1.0.0-pre",
```